kurtosis-devnet/optimism-package-trampoline/kurtosis.yml

```diff
@@ -2,7 +2,7 @@ name: github.com/ethereum-optimism/optimism/kurtosis-devnet/optimism-package-tra
 description: |-
   A trampoline package for optimism-package. This one is reproducible, due to the replace directives below.
 replace:
-  github.com/ethpandaops/optimism-package: github.com/ethpandaops/optimism-package@68d71d7c2ca21a1541199ab34336fb03ae11d282
+  github.com/ethpandaops/optimism-package: github.com/sebastianst/optimism-package@719a32baf203fff384e4e5f6d7920e1471c987bd
```

Comment:
cc @janjakubnanista we need your help here to keep the main optimism package in line with the monorepo.